### PR TITLE
remove the support for docker options passed after --

### PR DESCRIPTION
**Pull request by eyal-resonai**
remove the support for docker options passed after -- (to allow executable which uses --)

Instead one can set the env variable YBT_DOCKER_OPTS to add args to the docker command

### CODE REVIEW

File: yabt/ybtbin.sh.tmpl [hunk 1]
```diff
@@ -1,33 +1,12 @@
 #!/bin/bash
 
-# Split command line on `--`
-# Anything before the first `--` is passed as args to the container command
-# Anything after the first `--` is passed as extra Docker options
-# The `--` must be surrounded by whitespace
+# to add more args to the docker command set a var DOCKER_OPTS:
```
Comment (itamaro) [inline yabt/ybtbin.sh.tmpl:3]: you mean YBT_... ?

```suggestion
# to add more args to the docker command set a var YBT_DOCKER_OPTS:
```

**Review by itamaro — APPROVED** [review]
lgtm

please make sure Travis is happy before merging.

not sure if it's still about six, or a new pytest version that changed the API or what

**Comment by eyal-resonai** [comment]
> lgtm
> 
> please make sure Travis is happy before merging.
> 
> not sure if it's still about six, or a new pytest version that changed the API or what

maybe that's the issue. opened a PR to replace pep8 with flake8. we'll see

**Comment by eyal-resonai** [comment]
Also force newer six

**Pull request merged**